.evergreen/scripts/generate_config.py

```diff
@@ -942,6 +950,26 @@ def create_enterprise_auth_tasks():
     return [EvgTask(name=task_name, tags=tags, commands=[server_func, assume_func, test_func])]
 
 
+def create_perf_tasks():
+    tasks = []
+    for version, ssl, sync in product(["6.0", "8.0"], ["ssl", "nossl"], ["sync", "async"]):
```

Comment:
We can remove the 6.0 tests now that we are pinning to v8.0-perf.

Reply:
Done